contrib/pkg/clusterpool/clusterpool.go

```diff
@@ -0,0 +1,464 @@
+package clusterpool
+
+import (
+	"fmt"
+	"os"
+	"os/user"
+	"path/filepath"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
Seperate k8s imports.
```suggestion
	"github.com/spf13/cobra"
	
	corev1 "k8s.io/api/core/v1"
```